### PR TITLE
fix: display sponsors

### PR DESCRIPTION
**Pull request by mmetesreau**
Managing rows

### CODE REVIEW

File: themes/hugo-agency-theme/layouts/partials/sponsors.html [hunk 1]
```diff
@@ -7,17 +7,23 @@ <h2 class="section-heading">{{ with .Site.Params.sponsors.title }}{{ . | markdow
                 <h3 class="section-subheading text-muted">{{ with .Site.Params.sponsors.subtitle }}{{ . | markdownify }}{{ end }}</h3>
             </div>
         </div>
-        <div class="row">
-            {{ range .Site.Params.sponsor }}
+        {{ $end := 4 }}
```
Comment (ldez) [inline themes/hugo-agency-theme/layouts/partials/sponsors.html]: could you use a better name.

Comment (mmetesreau) [reply themes/hugo-agency-theme/layouts/partials/sponsors.html]: Changed to $indexNextClosingTag

**Comment by ldez** [comment]
the problem is still here:
```
9:46:53 AM: Executing user command: hugo
9:46:53 AM: ERROR 2019/07/04 07:46:53 Failed to add template "theme/partials/sponsors.html" in path "/opt/build/repo/themes/hugo-agency-theme/layouts/partials/sponsors.html": template: theme/partials/sponsors.html:25: unexpected "=" in operand
9:46:53 AM: ERROR 2019/07/04 07:46:53 theme/partials/sponsors.html : template: theme/partials/sponsors.html:25: unexpected "=" in operand
```

**Comment by mmetesreau** [comment]
It looks like a Go version [issue](https://github.com/gohugoio/hugo/issues/5563). I don't have this error and it's the first time I write Go code... Should I use ```:=``` instead of ```=```?

**Comment by ldez** [comment]
nope, I will try to add a configuration to update the go version used by Netlify.

**Comment by ldez** [comment]
I updated the Hugo version on Netlify :wink:

**Pull request merged**